src/gateway.rs

```diff
@@ -226,7 +226,7 @@ impl Gateway {
                             None => warn!(logger, "packet transmitted with adjusted power, but packet forwarder does not indicate power used."),
                             Some(actual_power) => {
                                 info!(logger, "beacon transmitted"; "beacon" => &beacon_id, "power" => actual_power, "tmst" => tmst);
-                                warn!(logger, "packet transmitted with adjusted power.");
+                                debug!(logger, "packet transmitted with adjusted power.");
```

Comment:
This is still not useful.. the power field already indicates that a beacon was transmitted with a given power.. The "packet transmitted" just adds confusion

Reply:
Perhaps the phrasing itself could be improved, but I'm wary of a scenario where a packet forwarder does not behave the way we anticipate as we know quite a few vendors wrote their own. Adding a debug line to differentiate this code path from the "transmitted without adjustment" code path seems useful to me as it provides a little bit more visibility.

Reply:
then change the info! above it?

Reply:
> then change the info! above it?

ok